site/content/en/docs/drivers/includes/vmware_windows_usage.inc

```diff
@@ -0,0 +1,13 @@
+## Requirements
+
+*VMware Workstation
+
+## Driver Installation
+
+First install the [VMware Workstation](https://www.vmware.com/in/products/workstation-pro/workstation-pro-evaluation.html) 
+
+After installation, start a cluster using the vmware driver:
+
+```shell
+minikube start 
```

Comment:
we probably want `--driver vmware` here, because minikube will default to using docker if it's installed